install

```diff
@@ -253,13 +253,17 @@ unless mac?
   HOMEBREW_REPOSITORY = "#{HOMEBREW_PREFIX}/Homebrew".freeze
 end
 
+# Invalidate sudo timestamp before exiting (if it wasn't active before).
+at_exit { Kernel.system "/usr/bin/sudo", "-k" } if @have_sudo
+
+
 # Tests will fail if the prefix exists, but we don't have execution
 # permissions. Abort in this case.
 abort <<-EOABORT if File.directory?(HOMEBREW_PREFIX) && (!File.executable? HOMEBREW_PREFIX)
 The Homebrew prefix, #{HOMEBREW_PREFIX}, exists but is not searchable. If this is
 not intentional, please restore the default permissions and try running the
 installer again:
-    sudo chmod 775 #{HOMEBREW_PREFIX}
+    sudo chmod 775 $USER #{HOMEBREW_PREFIX}
```

Comment:
What's the purpose of this change?

Reply:
Sorry, this is just a misunderstanding that I saw somehow `chmod` as `chown`.
`$USER` was removed.